2022-07-31 14:11:56,419 INFO    MainThread:20164 [wandb_setup.py:_flush():76] Configure stats pid to 20164
2022-07-31 14:11:56,420 INFO    MainThread:20164 [wandb_setup.py:_flush():76] Loading settings from C:\Users\znk\.config\wandb\settings
2022-07-31 14:11:56,420 INFO    MainThread:20164 [wandb_setup.py:_flush():76] Loading settings from c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\settings
2022-07-31 14:11:56,420 INFO    MainThread:20164 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'mode': 'offline'}
2022-07-31 14:11:56,420 INFO    MainThread:20164 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-07-31 14:11:56,421 INFO    MainThread:20164 [wandb_init.py:_log_setup():461] Logging user logs to c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\offline-run-20220731_141156-mie2pvlc\logs\debug.log
2022-07-31 14:11:56,421 INFO    MainThread:20164 [wandb_init.py:_log_setup():462] Logging internal logs to c:\Users\znk\Desktop\Programme\image-processing-by-dl\segmentation\U-Net\wandb\offline-run-20220731_141156-mie2pvlc\logs\debug-internal.log
2022-07-31 14:11:56,421 INFO    MainThread:20164 [wandb_init.py:init():495] calling init triggers
2022-07-31 14:11:56,421 INFO    MainThread:20164 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {}
2022-07-31 14:11:56,421 INFO    MainThread:20164 [wandb_init.py:init():514] re-initializing run, found existing run on stack: 3dmf6u2d
2022-07-31 14:11:56,422 INFO    MainThread:20164 [wandb_run.py:_finish():1680] finishing run hubmap-organ-segmentation/3dmf6u2d
2022-07-31 14:11:56,424 ERROR   MainThread:20164 [jupyter.py:save_history():427] Run pip install nbformat to save notebook history
2022-07-31 14:11:56,424 INFO    MainThread:20164 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-07-31 14:11:56,424 INFO    MainThread:20164 [wandb_init.py:_jupyter_teardown():393] cleaning up jupyter logic
2022-07-31 14:11:56,424 INFO    MainThread:20164 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 0
2022-07-31 14:11:56,425 INFO    MainThread:20164 [wandb_run.py:_restore():1834] restore
2022-07-31 14:11:57,340 INFO    MainThread:20164 [wandb_run.py:_on_finish():1991] got exit ret: 
2022-07-31 14:11:57,442 INFO    MainThread:20164 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
local_info {
}

2022-07-31 14:11:58,792 INFO    MainThread:20164 [wandb_run.py:_footer_history_summary_info():3099] rendering history
2022-07-31 14:11:58,792 INFO    MainThread:20164 [wandb_run.py:_footer_history_summary_info():3131] rendering summary
2022-07-31 14:11:58,803 INFO    MainThread:20164 [wandb_init.py:init():548] starting backend
2022-07-31 14:11:58,804 INFO    MainThread:20164 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2022-07-31 14:11:58,808 INFO    MainThread:20164 [backend.py:ensure_launched():217] starting backend process...
2022-07-31 14:11:58,961 INFO    MainThread:20164 [backend.py:ensure_launched():222] started backend process with pid: 20668
2022-07-31 14:11:58,962 INFO    MainThread:20164 [wandb_init.py:init():558] backend started and connected
2022-07-31 14:11:58,964 INFO    MainThread:20164 [wandb_run.py:_label_probe_notebook():1084] probe notebook
2022-07-31 14:11:58,964 INFO    MainThread:20164 [wandb_run.py:_label_probe_notebook():1094] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-07-31 14:11:58,964 INFO    MainThread:20164 [wandb_init.py:init():634] updated telemetry
2022-07-31 14:11:59,052 INFO    MainThread:20164 [wandb_init.py:init():700] starting run threads in backend
2022-07-31 14:12:04,060 INFO    MainThread:20164 [wandb_run.py:_console_start():1893] atexit reg
2022-07-31 14:12:04,061 INFO    MainThread:20164 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.WRAP
2022-07-31 14:12:04,062 INFO    MainThread:20164 [wandb_run.py:_redirect():1803] Wrapping output streams.
2022-07-31 14:12:04,063 INFO    MainThread:20164 [wandb_run.py:_redirect():1827] Redirects installed.
2022-07-31 14:12:04,063 INFO    MainThread:20164 [wandb_init.py:init():732] run started, returning control to user process
